### DISCUSSION TITLE
Volumes menu inside the EFB

### DISCUSSION THREAD

**Discussion by VitorAls** [magknight/787aviatorsTracker#825] [Ideas]
Is it possible to create a tab in the EFB to adjust the volumes, motor, APU, ground power? Because whenever I fly with the 787 I need to reduce the X-Plane sounds almost to the minimum. Because it's deafening when the sounds are at 100%. As with many other aircraft I think it would be great to bring this to the 787.
![B789 - 2022-05-05 07 18 54](https://user-images.githubusercontent.com/104434415/166904564-1f20f620-c4fd-45fc-b158-26e210cfc8a2.png)

**Comment by aeroniemi** [comment]
just use the x-plane menu?
i've never head anyone complaining that the sound is *too loud*, normally it's people complaining that its too quiet
